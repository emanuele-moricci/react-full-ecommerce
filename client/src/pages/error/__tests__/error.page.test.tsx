import renderer from "react-test-renderer";

import Error from "src/pages/error/error.page";

describe("<Error />", () => {
  it("Should match the snapshot", () => {
    const component = renderer.create(<Error />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
