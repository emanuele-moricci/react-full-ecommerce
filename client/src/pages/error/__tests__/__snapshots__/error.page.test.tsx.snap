// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> Should match the snapshot 1`] = `
<div
  className="sc-bdvvtL fvTUjG"
>
  <div
    className="sc-gsDKAQ tTmTM"
  />
  <h2
    className="sc-dkPtRN dVpXNZ"
  >
    Oops, looks like something went wrong...
  </h2>
</div>
`;
